vagrant/openshift/roles/client/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+- name: install packages
+  yum: name={{ item }} state=present
+  with_items:
+    - epel-release
+    - centos-release-gluster
+    - wget
+    - git
+    - net-tools
+    - bind-utils
+    - iptables-services
+    - bridge-utils
+    - bash-completion
+    - dnsmasq
+    - ncurses-term
+
+- name: Add paas SIG repository
+  yum_repository:
+    name: paas-openshift-origin
+    description: PaaS openshift-origin repo
+    baseurl: http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/
+    gpgkey: https://raw.githubusercontent.com/openshift/openshift-ansible/master/roles/openshift_repos/files/origin/gpg_keys/openshift-ansible-CentOS-SIG-PaaS
+    gpgcheck: yes
+
+- name: install ansible and Heketi utilities
+  yum: name={{ item }} state=present
+  with_items:
+    - ansible-2.2.1.0-2.el7
```

Comment:
ansible-2. 2.1.0-2.el7 does not exist and playbook fails - changing it as ansible works